frontend-react/src/utils/CustomRenderUtils.tsx

```diff
@@ -40,54 +41,80 @@ export const makeOktaHook = (_init?: Partial<IOktaContext>): OktaHook => {
     React error when rendering for unit tests.
 */
 const RouterWrapper: FC = ({ children }) => {
-    return <BrowserRouter>{children}</BrowserRouter>;
+    return (
+        <BaseWrapper>
+            <BrowserRouter>{children}</BrowserRouter>
+        </BaseWrapper>
+    );
 };
 
 export const SessionWrapper =
     (mockOkta: OktaHook) =>
     ({ children }: PropsWithChildren<{}>) => {
         return (
-            <RouterWrapper>
-                <SessionProvider oktaHook={mockOkta}>
-                    {children}
-                </SessionProvider>
-            </RouterWrapper>
+            <BaseWrapper>
+                <RouterWrapper>
+                    <SessionProvider oktaHook={mockOkta}>
+                        {children}
+                    </SessionProvider>
+                </RouterWrapper>
+            </BaseWrapper>
         );
     };
 
 export const QueryWrapper =
     (client: QueryClient = new QueryClient()) =>
     ({ children }: PropsWithChildren<{}>) =>
         (
-            <QueryClientProvider client={client}>
-                <AuthorizedFetchProvider initializedOverride={true}>
-                    {children}
-                </AuthorizedFetchProvider>
-            </QueryClientProvider>
+            <BaseWrapper>
+                <QueryClientProvider client={client}>
+                    <AuthorizedFetchProvider initializedOverride={true}>
+                        {children}
+                    </AuthorizedFetchProvider>
+                </QueryClientProvider>
+            </BaseWrapper>
         );
 
+export const BaseWrapper = ({ children }: PropsWithChildren<{}>) => (
```

Comment:
Honestly, I feel like it'd simplify a lottttt if we just make one big wrapper that includes all Providers with sensible default wherever needed.  I mean, that's how every literally every component is rendered on our site so I can't imagine that this'd be too problematic, but I've learned to not make assumptions in this codebase 😆 

What are your thoughts about this, though?  No changes needed here -- just thinking aloud!

Reply:
I agree and am in favor of going that route if someone with better testing experience than I sees that as best.